src/mesh/boundary_standard.cxx

```diff
@@ -3197,7 +3197,7 @@ void BoundaryFree_O3::apply_ddt(Field3D &f) {
 ///////////////////////////////////////////////////////////////
 
 BoundaryOp* BoundaryRelax::cloneMod(BoundaryOp *operation, const std::list<std::string> &args) {
-  BoundaryRelax* result = new BoundaryRelax(operation, r);
+  auto* result = new BoundaryRelax(operation, r);
```

Comment:
`unique_ptr`?

Reply:
Not these boundary ones, as we're returning `result`, so we no longer own it. Really, these should return `unique_ptr`, but that will have to wait till The Great Boundary Refactor